modules/helm-filtering-helm-charts-by-certification-level.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/configuring-custom-helm-chart-repositories.adoc
+
+[id="filtering-helm-charts-by-certification-level_{context}"]
+= Filtering Helm Charts by their certification level
+
+As a user, you can use the source filter to filter the Helm charts based on their certification level in the *Developer Catalog*.
+
+.Procedure
+
+. In the *Developer perspective*, navigate to the *+Add* view and select a project. From the *Developer Catalog*, select the *Helm Chart* option to see all the Helm charts in the *Developer Catalog*.
+
+. Click on the Source filter in the Helm catalog to filter the Helm charts based on provider type.
+
+[NOTE]
+====
+Source filter will not be visible when there is only one provider type.
+====
+
+. Select a chart and read the description, README, and other details about the chart. A certified badge can also be viewed with the Helm chart title in the catalog page and details panel for some of the helm charts.
```

Comment:
Replace with: You can now select the required chart and install it. This should be the ending statement of the module, not a step.

Reply:
Done!